src/irmin-pack/conf.mli

```diff
@@ -68,16 +81,43 @@ module Key : sig
 end
 
 val fresh : Irmin.Backend.Conf.t -> bool
+(** Flag to indicate that the store will start with fresh data on disk. Warning:
+    setting this to [true] will delete existing data. Default is [false]. *)
+
 val lru_size : Irmin.Backend.Conf.t -> int
+(** Size, in number of entries, of LRU cache. Default [100_000]. *)
+
 val index_log_size : Irmin.Backend.Conf.t -> int
+(** Size, in bytes, of index log. Default [2_500_00]. *)
```

Comment:
Is this the log cache? Or what is on disk?

Reply:
I think this is the on-disk size of the index's log.